modules/machineconfig-modify-dns-nameservers.adoc

```diff
@@ -0,0 +1,174 @@
+[source,terminal]
+----
+W21haW5dCmRucz1ub25lCg==
+----
+
+. Create the `MachineConfig` object file. Replace the base64 strings with the values from the previous steps.
+This example adds the machine configuration to the nodes with the `master` role.
+You can change it to `master` or `worker` or make a separate `MachineConfig` object for the each role:
++
+[source,terminal]
+----
+$ cat << EOF > ./override-etc-resolv-conf.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: override-etc-resolv-conf
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64, IyBDdXN0b20gcmVzb2x2LmNvbmYgYWRkZWQgd2l0aCBNYWNoaW5lIENvbmZpZyBPcGVyYXRvcgojIGFuZCB0aGUgb3ZlcnJpZGUtZXRjLXJlc29sdi1jb25mLnlhbWwgZmlsZS4Kc2VhcmNoIHN2Yy5jbHVzdGVyLmxvY2FsIGNsdXN0ZXIubG9jYWwgZXhhbXBsZS5jb20KbmFtZXNlcnZlciAxOTIuMTY4LjI0LjEKbmFtZXNlcnZlciAxMC4wLjAuMQpuYW1lc2VydmVyIDEwLjAuMC4yCg==
+        mode: 420
+        overwrite: true
+        path: /etc/resolv.conf
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W21haW5dCmRucz1ub25lCg==
+        mode: 420
+        overwrite: true
+        path: /etc/NetworkManager/conf.d/disable-network-manager-dns.conf
+EOF
+----
+
+. Make a backup copy of the configuration file.
+
+. Apply the configuration in one of two ways:
++
+* If the cluster is not running, generate manifest files, add this file to the `openshift` directory, and continue to create the cluster.
++
+* If the cluster is running, apply the file as follows:
++
+[source,terminal]
+----
+ $ oc apply -f ./override-etc-resolv-conf.yaml
+----
+
+.Verification
+
+. Monitor the progress of the Machine Config Operator by running the following command:
++
+[source,terminal]
+----
+$ oc get machineconfigpool
+----
++
+.Expected output
+[source,terminal]
+----
+NAME   CONFIG             UPDATED UPDATING DEGRADED MACHINECOUNT READYMACHINECOUNT UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT AGE
+master rendered-master-35 True    False    False    3            3                 3                   0                    34m
+worker rendered-worker-d8 False   True     False    3            1                 1                   0                    34m
+----
++
+Wait until the output from the command reports `True` for master and worker nodes in the `UPDATED` column.
+
+. Display the contents of the `/etc/resolv.conf` file on a host.
+
+.. Store the host name for one worker node in a variable:
++
+[source,terminal]
+----
+$ export ONE_HOST=$(oc get node -l node-role.kubernetes.io/worker="" \
+    -o jsonpath="{.items[0].metadata.name}")
+----
+
+.. Display the file contents from the worker node. As an alternative, you can specify `oc debug node/<host-name>` for each host in your cluster:
++
+[source,terminal]
+----
+$ oc debug node/${ONE_HOST} -- chroot /host cat /etc/resolv.conf
```

Comment:
I don't know if it's helpful to users or not, but I've always defined any variables used in a command, even if it ought to be obvious:

```asciidoc
[source,terminal]
----
$ oc debug node/${ONE_HOST} -- chroot /host cat /etc/resolv.conf
----
+
where:
+
`ONE_HOST`:: Specifies the host name captured in the previous command.
```